cmd/mysterium_monitor/command_run/command.go

```diff
@@ -43,22 +43,19 @@ func (cmd *CommandRun) Run(options CommandOptions) error {
 		return errors.New("Failed to get original IP: " + err.Error())
 	}
 
-	cmd.clientCommand = command_client.NewCommand(
-		state_client.NewMiddleware(cmd.checkClientIpWhenConnected),
-	)
+	cmd.clientCommand, err = command_client.NewCommand(command_client.CommandOptions{
```

Comment:
Would be good not to disconnect middleware `state_client.NewMiddleware(cmd.checkClientIpWhenConnected)`

Reply:
I think this client api checking needs reworking anyway, as it's not practical to pass low level VPN spefic middlewares to client just for testing purposes. Since a lot has changed, how client setups VPN connection. Maybe we should write a simple functionality which setups vpn connection by using tequilapi and then checks client's outgoing ip address?

Reply:
This will be fixed later with mysterium_monitor